### PR TITLE
SWI-3362 Rename Package to `com.bandwidth.sdk` 

### PR DESCRIPTION
**Pull request by ajrice6713**
@hamermike is this going to be an issue? The current package is named `com.bandwidth` - i think `com.bandwidth.sdk` is more appropriate

### CODE REVIEW

File: .github/workflows/maven.yml
Comment (ckoegel) [inline .github/workflows/maven.yml:1]: this got reverted again

File: .DS_Store
Comment (ckoegel) [inline .DS_Store:1]: should this be here?

File: .gitignore
Comment (ckoegel) [inline .gitignore:1]: is this supposed to remove that? I thought we want `.idea` in gitignore?

File: src/main/java/com/bandwidth/sdk/model/AnswerCallback.java
Comment (ckoegel) [inline src/main/java/com/bandwidth/sdk/model/AnswerCallback.java:1]: this file

File: .DS_Store
Comment (hamermike) [reply .DS_Store:1]: No

File: .openapi-generator-ignore
Comment (ckoegel) [inline .openapi-generator-ignore:1]: is the generator still going to overwrite the gitignore and remove the .DS_Store now?

Comment (ajrice6713) [reply .openapi-generator-ignore:1]: fixed

**Review by ckoegel — APPROVED** [review]

**Review by juliajanu — APPROVED** [review]

**Pull request merged**